### PR TITLE
Feature/BulkAssignment - Select all

### PR DESCRIPTION
**Pull request by petrKavulok**
### in this PR

- `Select all` button in Credentials Selection Card
- Clear selection button in Selected Credentials Card
- Missing locales update

https://github.com/TeskaLabs/seacat-admin-webui/assets/79644454/a6fae487-45df-4a50-8133-2b9302a1e51a

### CODE REVIEW

**Comment by petrKavulok** [comment]
https://github.com/TeskaLabs/seacat-admin-webui/assets/79644454/33543c0e-1184-4d84-a25c-541b785a973b

Screenshot 2023-07-20 at 14 10 34
Screenshot 2023-07-20 at 14 10 45

File: src/modules/auth/credentials/BulkAssignmentContainer.js [hunk 1]
```diff
@@ -306,12 +338,37 @@ const BulkAssignmentContainer = (props) => {
...
 
+			<div className='credentials-actions'>
+				<Button
+					primary
+					outline
+					onClick={() => bulkSelection('add')}
+					title={t("BulkAssignmentContainer|Select all displayed credentials")}
+					disabled={allSelected}
+				>
+					<span>
```
Comment (Pe5h4) [inline src/modules/auth/credentials/BulkAssignmentContainer.js]: @petrKavulok I dont think the span wrapper is necessary here

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
@petrKavulok Looks good, however I am not sure about the `>` and `<` in the cards (the button right next to the credential/tenant).  When I see it it makes me a bit confused since I am not sure what those buttons are supposed to do. The `+` and `-` would be better

**Comment by petrKavulok** [comment]
> @petrKavulok Looks good, however I am not sure about the `>` and `<` in the cards (the button right next to the credential/tenant). When I see it it makes me a bit confused since I am not sure what those buttons are supposed to do. The `+` and `-` would be better

I agree with you, however these changes were requested, approved and blessed by Aleš

**Comment by Pe5h4** [comment]
> > @petrKavulok Looks good, however I am not sure about the `>` and `<` in the cards (the button right next to the credential/tenant). When I see it it makes me a bit confused since I am not sure what those buttons are supposed to do. The `+` and `-` would be better
> 
> I agree with you, however these changes were requested, approved and blessed by Aleš

Alrighty

**Review by Pe5h4 — APPROVED** [review]

**Review by ateska — APPROVED** [review]

**Pull request merged**